src/display.jl

```diff
@@ -23,20 +26,70 @@ script_content(p::Plot) = """
     Plotly.plot(thediv, data,  layout, {showLink: false});
     """
 
-stringmime(::MIME"text/html", p::Plot) =  """
-    <html>
-    <head>
-        <script src="$(_js_path)"></script>
-    </head>
 
-    <body>
-      $(html_body(p))
-    </body>
-    </html>
+function stringmime(::MIME"text/html", p::Plot, js::Symbol=:local)
+
+    if js == :local
```

Comment:
I think instead of repeating the whole html template for each possible `js` we should do something like this:

```julia
function stringmime(::MIME"text/html", p::Plot, js::Symbol=:local)
    if js == :local
        script_txt = "<script src=\"$(_js_path)\"></script>"
    elseif js == :remote
        script_txt = "<script src=\"$(_js_cdn_path)\"></script>"
    elseif js == :embed
        script_txt = "<script>$(readall(js_path))</script>"
    else
        msg = """
        Unknown value for argument js: $js. 
        Possible choices are `:local`, `remote`, `:embed`
        """
        throw(ArgumentError(msg))
    end
    """
    <html>
    <head>
       $script_txt
    </head>
    <body>
          $(html_body(p))
    </body>
    </html>
    """ 
end
```